watertap/ui/fsapi.py

```diff
@@ -611,6 +631,14 @@ def __init__(
             self.add_action("diagram", None)
 
         self._actions["custom_do_param_sweep_kwargs"] = custom_do_param_sweep_kwargs
+        self.fs_exp.add_option(
+            name="NumParallelWorkers",
+            display_name="Number of multi-processing workers",
```

Comment:
It might be more clear to say "Number of multi-processing workers for sensitivity analysis" 

right now when there are build options, we don't have a way to do any groupings. Might be nice if we had a groups for build options like we do for flowsheet inputs (e.g. flowsheet model build options, analysis options etc..)

Reply:
I think @MichaelPesce mentioned that he would exclude this bit from the PR